src/apiCalls.js

```diff
@@ -1,12 +1,21 @@
 const getAllQuestions = () => {
-  return fetch('http://localhost:8080/api/v1/questions')
-    .then(response => {
-      if(!response.ok) {
-        throw new Error('Server Down')
-      }
-      return response.json()
-    }) 
+  return fetch('http://localhost:8080/api/v1/questions').then(response => {
+    if (!response.ok) {
```

Comment:
Love this error handling! Apply catch blocks to your fetchs as well for more code safety!